ci-operator/step-registry/rosa/setup/readiness/cluster/rosa-setup-readiness-cluster-commands.sh

```diff
@@ -6,18 +6,20 @@ set -o pipefail
 
 trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
 
-
-HOSTED_CP=${HOSTED_CP:-false}
```

Comment:
How about OCP jobs? Are they using the parameters? Will it affect if we replace the parameters?

Reply:
It will not affect other jobs